frontend/src/index.tsx

```diff
@@ -197,6 +203,22 @@ const Store: Component = () => {
     db.remove_webxdc(app_id)
   }
 
+  const supportsDraggingOut = !!(window as any).webxdc_custom?.desktopDragFileOut
+  const onDragStart = async (ev: DragEvent, item: AppInfoWithState) => {
```

Comment:
I'd prefer to use a normal function definition (async function ()) here because that is also the way it's done for the other handlers.

Reply:
does not make a difference, feel free to change everything to your code style, you can checkout the desktop pr to test it.